Makefile.am

```diff
@@ -34,7 +34,9 @@ libcrun_SOURCES = src/libcrun/utils.c \
 			src/libcrun/chroot_realpath.c \
 			src/libcrun/signals.c \
 			src/libcrun/seccomp_notify.c \
-			src/libcrun/kontain.c
+			src/libcrun/kontain.c \
+			src/libcrun/kontain_nonkmexec.c \
+			src/libcrun/sha256.c
```

Comment:
See https://github.com/kontainapp/km/issues/943. No need to change anything now, le't have it merged first (and handle other issues in the current milestone) - the # 943 is in the Next milestone